identity/api.go

```diff
@@ -196,7 +195,7 @@ func (i *Identity) popAuth(au *Authenticate, atts []kyber.Point) (*CreateIdentit
 			break
 		}
 	}
-	sigtag := anon.Sign(i.Client.Suite().(anon.Suite), random.Stream, au.Nonce,
+	sigtag := anon.Sign(i.Client.Suite().(anon.Suite), i.Client.Suite().RandomStream(), au.Nonce,
```

Comment:
Why is the randomstream still here? I thought we moved that to suite...

same on line 108.

Reply:
Fixed.